fast-rpc-openapi-spec.json

```diff
@@ -0,0 +1,828 @@
+{
+  "openapi": "3.0.0",
```

Comment:
Not sure if openapi spec is the right way to document the RPC methods. The JSON-RPC format is different from REST format and rendering of openapi spec doesnt make much sense.

For eg. the path is not the way to send the request. The request is sent to the empty path and the method is part of the request body. Also each RPC has specific parameters that it accepts. Having generic JsonRpcRequest and JsonRpcResponse is not really helpful. Users need to know which parameters are accepted.

I am not entirely sure how to do this, but I was looking online and [found](https://openethereum.github.io/JSONRPC-eth-module#eth_estimategas). It would be better to just have a simple markdown document listing all the methods with the input and output listed out.